package Exception_class;

public class Man {
    public void Man(){
        System.out.println("创建一个人！");
    }
}
